Automattic/jetpack
# Products: update plans and product icons

#### Changes proposed in this Pull Request:

* Update Plans and Product icons.

Analog Calypso PR: https://github.com/Automattic/wp-calypso/pull/64144

#### Jetpack product discussion

pbNhbs-2GY-p2

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:

* Fire up this PR
* Use a site with a few active plans or products.
* Go to My Plan and ensure the new icons are visible.

#### Screenshots

Before | After
--|--
![image](https://user-images.githubusercontent.com/390760/170978037-eaffb405-7b81-4e3b-9912-236649395394.png)| ![image](https://user-images.githubusercontent.com/390760/170978057-a5462363-f31c-4a12-85f3-406131aaf5f5.png)
